calendar_event_report/tests/test_weekly_report_wizard.py

```diff
@@ -0,0 +1,146 @@
+# © 2019 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/LGPL).
+
+from odoo.tests import common
+import datetime
+
+
+class TestWeeklyReportWizard(common.TransactionCase):
+
+    def setUp(self):
+        super(TestWeeklyReportWizard, self).setUp()
+        self.Rooms = self.env['resource.calendar.room']
```

Comment:
Replace with reource.resource of calendar_resource, to remove project_resource_caelndar dep